spec/models/user_sign_out_event_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'yaml'
```

Comment:
We don't need this as far as I can tell... same with the sign_in spec from previous PR